source/deployment-options/offline-installation.rst

```diff
@@ -22,13 +22,24 @@ Prerequisites
 Download the packages and configuration files
 ---------------------------------------------
 
-#. Replace ``<deb|rpm>`` in the following command with your choice of package format and run it from a Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures.
+#. Run the following commands with your choice and execute them from a Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures.
 
    .. code-block:: console
 
       # curl -sO https://packages.wazuh.com/|WAZUH_CURRENT_MINOR|/wazuh-install.sh
       # chmod 744 wazuh-install.sh
-      # ./wazuh-install.sh -dw <deb|rpm>
+    
+   If your system is RPM-based, run the following command:
```

Comment:
We can change this section to use tabs for RPM and Debian options.

Reply:
Done in https://github.com/wazuh/wazuh-documentation/pull/6686/commits/bae4dcc84a08e98181f4c81dcb125d83cc0c0596.